packages/main/src/index.ts

```diff
@@ -49,6 +51,18 @@ app.on('window-all-closed', () => {
   }
 });
 
+app.on('before-quit', () => {
+  if (extensionLoader instanceof ExtensionLoader) {
```

Comment:
not sure why we need instanceof there ?
I would guess we only need `if (extensionLoader) {`

Reply:
or `extensionLoader?.stopAllExtensions`

Reply:
Thank you for the suggestion. I'll make the change.